package.json

```diff
@@ -53,4 +54,4 @@
     "nodemon": "^2.0.20",
     "prettier": "^3.0.3"
   }
-}
+}
```

Comment:
can you add a final linebreak back to this file? I guess eslint doesn't check the package.json-- maybe that's possible in the future with some better configuration

Reply:
Final linebreak added. Sorry about the ES Lint multi commits. I'm running it locally now before a push.